### ISSUE TITLE
'Ticking Memory Connection' crash upon entering world

### ISSUE THREAD

**Issue by Febilian** [BrassGoggledCoders/SteamCraft2#320]
https://paste.ee/p/p4Nzp

The error doesn't refer SC2 or Boilerplate specifically, but I only started having this problem after updating both of them in my private modset, so I can only guess that it's related to SC2 or Boilerplate somehow.

It happens when creating a new world, logging into an existing one, or trying to log into a server with the same version of SC2/Boilerplate installed. I don't know what other information I could come up with that would be of help, but if there's something else that I might be able to provide, please let me know.

Using build 91 of SC2 and build 42 of Boilerplate, both from the Jenkins repository.

Reverting to build 90 of SC2 and its associated Boilerplate version causes the issue to disappear, further confirming my suspicion that it is related to this set of mods.

Did something change between build 90 and 91 that would affect existing worlds/items/IDs that weren't explicitly mentioned in the changes of the build, by the way? I have some HQM quests set up in my private modpack that involve Steamcraft 2 and if there's a chance that this issue is caused by HQM attempting to reference some items that aren't there, then I think it would've been worth bringing up.

**Comment by SkySom** [comment]
Does this happen with only SC2/Boilerplate installed? I don't know of any item/block changes... outside of maybe some where we changed some item internals.

**Comment by Febilian** [comment]
Just tested it with nothing but SC2/Boilerplate(and Optifine, sometimes I get weird graphic errors without it) and it launched just fine... whoops.

Looking back, I think it's quite likely that it might be an issue with HQM trying to reference something that doesn't properly exist anymore since the update, or that's changed somehow... the log(not the crash log, but the debug console from when the game runs in MultiMC) mentioned HQM and quest issues in errors, but I wasn't sure if that was an error that normally happens or not during the loading of the game that is actually harmless.

I apologize, I fear I've wasted your time with this. Wish I knew how to properly update while still maintaining the quests I've got from HQM, but I guess that's not something you guys would really help with, so I'll get out of your hair.
